### DISCUSSION TITLE
Is the repo ready to use for refactoring CORE-Azure?

### DISCUSSION THREAD

**Discussion by drewcdisc** [cdisc-org/cdisc-rules-engine#71] [General]
Aleksei asked an important question that I want to make public so that it can apply to other platforms as well. **When should we start refactoring the Azure version?** 

I think that question can be more generally asked: **Is this repo ready to be used as the base rules engine for other platforms?**

As of 11:00AM EST, August 15, 2022, the repo is private and so this discussion is private. Hopefully this will become public once we make the repo public. This is an important discussion.

Please, all devs on this release, take time to answer this question.

**Comment by nhaydel** [comment]
The repo is ready to be used as a rules engine for other platforms

**Comment by nhaydel** [comment]
@drewcdisc we would like to start refactoring CORE-Azure. We were planning on tracking CORE-Azure work in azure devops. Can  you set up a Kanban board for us to use?

**Reply by drewcdisc** [reply]
Will do

**Reply by drewcdisc** [reply]
@nhaydel and others: I will create a new project under ADO since Nick De Donder is going to be pulling feature information out of the original cdisc-CORE project to use in the new github-based zenhub project.

**Reply by drewcdisc** [reply]
I've created a project in ADO, Post-MVP-CORE, that is specifically for refactoring for the new cdisc-org/core-rules-engine.  I've left Github versioning on (it was the default).  I've invited Nic, Aleksei, Dmitry, and Denis.  I've used the Basic process to get started. It has To-Do, Doing, and Done columns.

**Reply by drewcdisc** [reply]
If you'd like to invite others, please let me know, or I believe you can all invite them yourselves. The project is Private so only Invitees can see the project.

**Reply by drewcdisc** [reply]
I've also made all of you members of the Build team. If that's too broad let me know and I can pull that back and add just those that you want.
